.packit.yaml

```diff
@@ -31,15 +35,32 @@ jobs:
 - job: propose_downstream
   trigger: release
   dist_git_branches:
-    - fedora-rawhide
+    - fedora-all
```

Comment:
Specifying all possible branches downstream is not good. PackIt will create a PR per branch and they will all be similar. An exception might be EPEL 8. I think EPEL 8 will need a separate job to create a PR with a spec file for RHEL8.

A subsequent fedora-release is done by merging the PR created by PackIt into the rawhide(main) branch, and all other branches are merged into rawhide(main). So all Fedoras and EPEL9 versions have the same content. Creating multiple PRs with the same content is not necessary.